docs/strategies/retry.md

```diff
@@ -359,16 +361,16 @@ await retry.ExecuteAsync(ct =>
 <!-- endSnippet -->
 
 **Reasoning**:
-- If the `DoSomething` and `BeforeEachRetry` coupled together then decorate them together
-  - Or create a simple wrapper to call them in the desired order
 
-### 6 - Having a single policy to cover multiple failures
+If `DoSomething` and `BeforeEachAttempt` are interdependent, group them or craft a simple wrapper to invoke them in the correct sequence.
 
-Lets suppose we have an `HttpClient` which issues a request and then we try to parse a large Json
+### 6 - Having a single strategy for multiple failures
 
-❌ DON'T
+Suppose we have an `HttpClient` that issues a request and then we try to parse a large JSON.
 
-Have a single policy to cover everything
+❌ **Don't**
```

Comment:
Why did you change `DON'T` to `**Don't**`?

Reply:
Fixed.